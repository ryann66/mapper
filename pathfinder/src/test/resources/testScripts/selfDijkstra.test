# Check pathfinding to current location

CreateGraph mg

AddNode mg A

FindPath mg A A

AddEdge mg A A 1.0

FindPath mg A A

AddNode mg B
AddNode mg C
AddEdge mg A B 1.0
AddEdge mg B A 1.0
AddEdge mg B C 1.0
AddEdge mg C A 1.0

FindPath mg A A