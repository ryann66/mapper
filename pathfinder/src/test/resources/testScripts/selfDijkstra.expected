# Check pathfinding to current location

created multigraph mg

added node A to mg

path from A to A:
total cost: 0.000

added edge 1.000 from A to A in mg

path from A to A:
total cost: 0.000

added node B to mg
added node C to mg
added edge 1.000 from A to B in mg
added edge 1.000 from B to A in mg
added edge 1.000 from B to C in mg
added edge 1.000 from C to A in mg

path from A to A:
total cost: 0.000